src/spyglass/utils/dj_merge_tables.py

```diff
@@ -533,13 +537,21 @@ def fetch_nwb(
             raise ValueError("Try replacing Merge.method with Merge().method")
         restriction = restriction or self.restriction or True
         merge_restriction = self.extract_merge_id(restriction)
-        sources = set((self & merge_restriction).fetch(self._reserved_sk))
+
+        if log_export and self.export_id:
+            self._log_fetch(restriction=merge_restriction)
+
+        sources = set(
+            (self & merge_restriction).fetch(
+                self._reserved_sk, log_export=False
+            )
+        )
         nwb_list = []
         merge_ids = []
         for source in sources:
             source_restr = (
                 self & {self._reserved_sk: source} & merge_restriction
```

Comment:
```suggestion
                self & dj.AndList([{self._reserved_sk: source}, merge_restriction])
```

This was getting picked up by export logging of restrictions and registering everything for the part table.